chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,7 +150,8 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
-            d.fixed and not d.claimed?
+            next unless d.fixed || d.multipath?
```

Comment:
Not sure what do you want me to explain, I think this is pretty straightforward? It just adds the checks for multipath to the existing checks, as the commit message explains